KenticoCloud.Delivery/DeliveryClient.cs

```diff
@@ -511,6 +522,27 @@ public DeliveryClient(string projectId, string previewApiKey) : this(new Deliver
                 throw new InvalidOperationException("Preview API and secured Delivery API must not be configured at the same time.");
             }
 
+            return _deliveryOptions.EnableResilienceLogic ? await GetRepeatableDeliverResponseAsync(endpointUrl) : await GetSolitaryDeliveryResponseAsync(endpointUrl);
+        }
+
+        private async Task<JObject> GetRepeatableDeliverResponseAsync(string endpointUrl)
+        {
+            var policyResult = await ResiliencePolicyProvider?.Policy?.ExecuteAndCaptureAsync(() =>
+                {
+                    return SendHttpMessage(endpointUrl);
+                }
+            );
+
+            return await GetResponseContent(policyResult?.FinalHandledResult ?? policyResult?.Result);
```

Comment:
wouldn't it make sense to call `GetResponseContent` and its other occurrence in the `private async Task<JObject> GetDeliverResponseAsync(string endpointUrl)` method? (just askin')

Reply:
Yes, it would. I see this as a matter of personal preference. :)

Reply:
Done.